pkg/minikube/audit/audit.go

```diff
@@ -121,7 +121,11 @@ func LogCommandEnd(id string) error {
 }
 
 func getStartIndex(entryCount int) int {
-	maxEntries := viper.GetInt(config.MaxAuditEntries)
+	// default to 1000 entries
```

Comment:
hm... I wonder what is the best thing to do to if the audilog was already more than that in the files...should delete it ?

Reply:
This truncated the existing file, so if it's currently 2000 lines, it will truncate it to the last 1000